plugins/get_plugins_script.py

```diff
@@ -1,21 +1,35 @@
 import json
 import requests
+from typing import List, Dict
 from os import makedirs
 from os.path import join, dirname, realpath, isdir
 
+REALPATH = dirname(realpath(__file__))
+
 MARKETPLACE = 'https://github.com/ONLYOFFICE/onlyoffice.github.io'
 BRANCH = 'tree/master'
 SDKJS_PLUGIN_SOURCE = 'sdkjs-plugins/content'
 
 URL = f"{MARKETPLACE}/{BRANCH}/{SDKJS_PLUGIN_SOURCE}"
-PLUGIN_LIST_PATH = join(dirname(realpath(__file__)),
-                        'plugins-list-actual.json')
+PLUGIN_LIST_PATH = join(REALPATH, 'plugins-list-actual.json')
+EXCLUDED_LIST_PATH = join(REALPATH, 'excluded-plugins-list.json')
+
+
+def read_json(path: str) -> List | Dict:
```

Comment:
List | Dict. it is better to write 'List | Dict' with quotation marks for compatibility with earlier versions of python